core/src/autogluon/core/models/ensemble/fold_fitting_strategy.py

```diff
@@ -425,6 +432,17 @@ def _ray_fit(
     logger.debug(f"head node: {is_head_node}")
     logger.debug(f"executing fold on node {node_id}")
     logger.log(10, "ray worker training")
+
+    try:
+        import torch
+        visible_gpus = os.environ.get("CUDA_VISIBLE_DEVICES", "not set")
+        num_gpus = torch.cuda.device_count()
+        current_gpu = torch.cuda.current_device() if torch.cuda.is_available() else "N/A"
+        print(f"[GPU DEBUG] CUDA_VISIBLE_DEVICES={visible_gpus}, Torch sees {num_gpus} GPUs, Using GPU {current_gpu}", flush=True)
+    except ImportError:
+        pass
+    except Exception as e:
+        print(f"[GPU DEBUG] Could not get GPU info: {e}", flush=True)
```

Comment:
This is too heavy of a debug statement, we don't want to import torch unless we have to. You can add a debug flag for this, but don't import torch by default in workers or it slows everything down.